test/gtest/ucp/ucp_test.cc

```diff
@@ -686,7 +686,7 @@ void *ucp_test_base::entity::disconnect_nb(int worker_index, int ep_index,
         return req;
     }
 
-    ASSERT_UCS_OK(UCS_PTR_STATUS(req));
+    /* close request can be completed with any status depends on peer state */
```

Comment:
ok for now, in next PR by @Artemy-Mellanox close(FORCE) will always return NULL (success)

Reply:
but close(flush) can be completed with err too

Reply:
yes, close (flush) can complete with err
can u pls open another PR with doc clarification on this:
e.g close(flush) operation can return failure if the remote side disconnected at same time, but nevertheless all resources associated with the endpoint will be destroyed.
close(force) will always succeed

Reply:
IMHO this way we add extra commitment without any benefit.
maybe don't differentiate flush/force return status and just mention that "close request can be completed with error if the remote side disconnected at same time but nevertheless all resources associated with the endpoint will be destroyed"?